qdev_wrappers/tests/dataset/test_doNd.py

```diff
@@ -163,3 +189,31 @@ def test_do2d_out_type(_parameters):
                  _param, _paramComplex)
 
     assert type(_data[0]) == int
+
+
+def test_do2d_data(_parameters):
+
+    _start_p1 = 0
```

Comment:
all identifiers that are in a private scope anyways don't need an additional `_` . Here e.g. `_start_p1` cannot be accessed from any other place than from within the test function.
For module level functions, like the fixtures you are defining, I feel that you are right if you are very strict about it, but it would also be ok to not have the `_` in the fixtures.